lib/src/compiler/code_generator.dart

```diff
@@ -5104,7 +5104,7 @@ String jsLibraryName(String buildRoot, LibraryElement library) {
         uri.path.substring(buildRoot.length).replaceAll('/', separator);
   } else {
     // We don't have a unique name.
-    throw 'Invalid build root.  $buildRoot does not contain ${uri.toFilePath()}';
+    throw 'Invalid build root. $buildRoot does not contain ${uri.toFilePath()}';
```

Comment:
fixed a long line here. This doesn't fix https://github.com/dart-lang/dev_compiler/issues/530 though.